R/step_hcai_missing.R

```diff
@@ -0,0 +1,144 @@
+#' @title
+#' Clean NA values from categorical/nominal variables
+#'
+#' @description `step_hcai_missing` creates a specification of a recipe that 
```

Comment:
Unfortunately backticks don't work for generating Rd files. It's fine if you want to leave it, or you could use `\code{xyz}`

Reply:
I'm so used to markdown these days!